templates/Dockerfile.template

```diff
@@ -80,21 +80,15 @@ RUN set -ex; \
 ENV PG_MAJOR {{ $postgresql.Major }}
 ENV PG_VERSION {{ $postgresql.Version }}
 
-{{- if (eq .Repo "postgresql9") }}
-RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ jessie-pgdg main' $PG_MAJOR > /etc/apt/sources.list.d/pgdg.list
-{{- else }}
 RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ stretch-pgdg main' $PG_MAJOR > /etc/apt/sources.list.d/pgdg.list
-{{- end }}
 
 RUN apt-get update \
 	&& apt-get install -y postgresql-common \
 	&& sed -ri 's/#(create_main_cluster) .*$/\1 = false/' /etc/postgresql-common/createcluster.conf \
 	&& apt-get install -y \
 		postgresql-$PG_MAJOR=$PG_VERSION \
 {{- if (eq .Repo "postgresql9") }}
-		postgresql-contrib-$PG_MAJOR=$PG_VERSION \
-{{- else }}
-		postgresql-contrib \
+                postgresql-contrib-$PG_MAJOR=$PG_VERSION \
```

Comment:
Please use tabs instead of spaces, this is specific to this file as indentations in this specific file are made by tabs.